content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -196,7 +196,7 @@ kubectl logs myapp-pod -c init-myservice # Inspect the first init container
 kubectl logs myapp-pod -c init-mydb      # Inspect the second init container
 ```
 
-At this point, those init containers will be waiting to discover Services named
+At this point, those init containers will be waiting to discover [Services](/docs/concepts/services-networking/service/) named
```

Comment:
This could be a glossary tooltip:
```suggestion
At this point, those init containers will be waiting to discover
{{< glossary_tooltip text="Services" term_id="service" >}} named
```